docs/user_guides/projects/jobs/notebook_job.md

```diff
@@ -0,0 +1,177 @@
+---
+description: Documentation on how to configure and execute a Jupyter Notebook job on Hopsworks.
+---
+
+# How To Run A Jupyter Notebook Job
+
+## Introduction
+
+All members of a project in Hopsworks can launch the following types of applications through a project's Jobs service:
+
+- Python (*Hopsworks Enterprise only*)
+- Apache Spark
+
+Launching a job of any type is very similar process, what mostly differs between job types is
```

Comment:
Launching a job is a similar process for Python/Spark/Flink, with the differences being
the various configuration parameters for each framework/language. After following this guide you will be able to create a job using a Jupyter Notebook.